Framework/DataHandling/src/LoadSampleShape.cpp

```diff
@@ -172,6 +45,9 @@ void LoadSampleShape::init() {
       make_unique<FileProperty>("Filename", "", FileProperty::Load, extensions),
       "The path name of the file containing the shape");
 
+  // scale to use for stl
+  declareProperty("Scale", "cm", "The scale of the stl: m, cm, or mm");
```

Comment:
If there is a reason centimetre is the default (over metre) it would be worth mentioning it.

Reply:
was just checking with martyn, and the main reason we defaulted to centimetres is that for the majority of use cases its likely that the sample or environment pieces will be on that scale.